express-static/index.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const app = express();
+const path = require('path');
+
+const directory = path.join('__dirname/public');
+const access = express.static('public');
```

Comment:
The absolute path that you created with `path.join()` should be used as the argument to `express.static()`